tools/base/BUILD

```diff
@@ -1,43 +1,14 @@
 load("@rules_python//python:defs.bzl", "py_binary")
 load("@base_pip3//:requirements.bzl", "requirement")
 load("//bazel:envoy_build_system.bzl", "envoy_package")
-load("//tools/base:envoy_python.bzl", "envoy_py_library")
 
 licenses(["notice"])  # Apache 2
 
 envoy_package()
 
-envoy_py_library(
-    "tools.base.aio",
-    deps = [
-        requirement("aio.functional"),
-    ],
-)
-
-envoy_py_library(
-    "tools.base.checker",
-    deps = [
-        ":runner",
-    ],
-)
-
-envoy_py_library(
-    "tools.base.runner",
-    deps = [
-        requirement("colorama"),
-        requirement("coloredlogs"),
-        requirement("frozendict"),
-        requirement("verboselogs"),
-    ],
-)
-
-envoy_py_library(
-    "tools.base.utils",
-    deps = [
-        requirement("pyyaml"),
-        requirement("setuptools"),
-    ],
-)
+exports_files([
+    "base_command.py",
```

Comment:
Where exactly is this file? I don't see it in this PR or in the repo as of https://github.com/envoyproxy/envoy/commit/5c4ff96652b449a39cc640a97794333b7bc723d4?

Reply:
good catch - this should be removed i think